server/src/users/EventRouter.ts

```diff
@@ -18,31 +18,113 @@ export const eventRouter = Express.Router();
 
 // GET requests/:id currently using the Name field (first name) since don't know how to actually grab object id in salesforce
 
+eventRouter.get("/", async (req: Express.Request, res: Express.Response) => {
+    const limit: number = req.query.limit as any;
+    const offset: number = req.query.offset as any;
+    try {
+        const fetchedEvents = await EventService.getAllEvents(limit, offset);
+        res.status(200).send(fetchedEvents);
+    } catch (e) {
+        res.status(404).send(e.message);
+    }
+});
+
+eventRouter.get('/applications', async (req: Express.Request, res: Express.Response) => {
+    const name: string = req.query.name as string;
+
+    console.log("Hit applications endpoint")
+
+    try {
+        if (name !== undefined) {
```

Comment:
better convention to check if value is truthy rather than explicitly checking for one of the not-truthy values.

i.e. replace `if (var !== undefined) {` with `if (value) {`

Reply:
Resolved this and other similar issues in the same file in e361081